client/src/components/Workbench/WorkbenchSidebar/WorkbenchSidebar.jsx

```diff
@@ -55,6 +55,12 @@ const WorkbenchSidebar = ({ disabled, showUIPatterns, loading }) => {
       hidden: hideApps,
       children: <HistoryBadge unread={unread} disabled={disabled} />,
     },
+    {
+      to: path + ROUTES.SYSTEM_STATUS,
+      label: 'System Status',
+      iconName: 'data-files',
+      disabled: disabled,
+    },
```

Comment:
I'm getting a console error that I'm not seeing on the tapis v3 branch that may originate here?

```
`react-jsx-dev-runtime.development.js:117 
Warning: Failed prop type: Invalid prop `loading` of type `number` supplied to `Sidebar`, expected `boolean`.
    at Sidebar (https://cep.test:3000/core/static/src/components/_common/Sidebar/Sidebar.jsx:91:3)
    at WorkbenchSidebar 
    at div
    at Workbench (https://cep.test:3000/core/static/src/components/Workbench/Workbench.jsx?t=1686783174990:44:7)
    at Route2 `
```

Reply:
Looks like this has been resolved